Adicionando 2685 e 2686 (#85)
petbccufscar/URI: Iniciante/2686/README.md

# Problema:
Novamente Júlio pede sua ajuda, ele esqueceu de um pequeno detalhe. Como o seu o programa anterior só informava uma saudação, ele pediu que transformasse o grau do Sol/Lua em HH:MM:SS. Então caso aceite: dado um grau relativo a posição do Sol/Lua, refaça o sistema só que agora além da saudação de cada período do dia, informe exatamente as horas, os minutos e segundos.

##### Problema completo: https://www.urionlinejudge.com.br/judge/pt/problems/view/2686

# Resolução:

> Esse exercício é uma continuação do exercício 2685. Seu onbjetivo é indicar em qual período do dia estamos e o horário com base no ângulo de inclinação do Sol/Lua. Se for entre 0 e 90° está de dia, entre 90 e 180° está de tarde, entre 180 e 270° é noite e entre 270 e 360° está de madrugada, como 360° está na mesma posição que 0° em uma circunferência, é considerado de manhã.

Reviewer: ```suggestion
Esse exercício é uma continuação do exercício 2685. Seu objetivo é indicar em qual período do dia estamos e o horário com base no ângulo de inclinação do Sol/Lua. Se for entre 0 e 90° está de dia, entre 90 e 180° está de tarde, entre 180 e 270° é noite e entre 270 e 360° está de madrugada, como 360° está na mesma posição que 0° em uma circunferência, é considerado de manhã.
```

Para começar, criamos quatro variáveis, sendo `m` e `tempo` do tipo `float` e `hora` e `minuto` do tipo `int`:
```c
    float m, tempo;
    int hora, minuto;
```
`m` representa o ângulo em graus que será usada para indicar o período do dia, `tempo` indica um valor auxiliar que será convertido em minutos, `hora` e `minuto` representa o tempo descrito pelo ângulo dado em horas e minutos, respectivamente. 

> O exercício indica que ele terá vários casos de teste, isso significa que haverão leituras até o final do arquivo [EOF - End of File](https://pt.wikipedia.org/wiki/EOF). Fazemos a leitura do `m` no `scanf` dentro do `while`:

Reviewer: ```suggestion
O exercício indica que ele terá vários casos de teste, significando que haverão leituras até o final do arquivo [EOF - End of File](https://pt.wikipedia.org/wiki/EOF). Fazemos a leitura do `m` no `scanf` dentro do `while`:
```

```c
    while (scanf("%f", &m) != EOF) {
```
Dentro do `while`, fazemos a leitura e comparamos ao EOF, portanto o `while` irá rodar até chegar no final do arquivo. Este é simbolizado pela quebra de linha (`\n`). Agora vamos verificar o ângulo dado e determinar o horário que ele representa:
```c
      if (m < 90) {
        printf("Bom Dia!!\n");
```

> Sabemos que o ângulo sempre será um valor entre 0 e 360 graus. Com isso verificamos se `m` é menor que 90 (`m < 90`) e, se for, mostramos a mensagem `Bom Dia!!`. Agora precisamos calcular a hora do dia:

Reviewer: ```suggestion
Sabemos que o ângulo sempre será um valor entre 0 e 360 graus. Com isso, verificamos se `m` é menor que 90 (`m < 90`) e, se for, mostramos a mensagem `Bom Dia!!`. Agora precisamos calcular a hora do dia:
```

```c 
        hora = 6;
        tempo = m/0.5;
        minuto = tempo*2;
            

>         if (minuto >= 60) {
>           hora = hora + (minuto/60);
>           minuto = minuto%60;
>         }
>         printf("%02d:%02d:00\n", hora, minuto);
>       }

Reviewer: ```suggestion

```

Como você coloca o mesmo trecho de código depois, achei que não precisaria aqui. Mas se preferir deixar tudo bem.

```

> Como está de dia, o horário deve estar entre 6 da manhã e meio-dia (12:00), por isso igualamos `hora` a 6. Em seguida calculamos quantos minuto a mais o ângulo indica. Analisando os casos de teste presentes no enunciado, descobrimos que 0,5° equivale a 2 minutos. Com isso, descobrimos quantos minutos temos dividindo `m` por 0,5 (`tempo = m/0.5`) e depois multiplicando o valor obtido por 2 (`minuto = tempo*2`). Agora resta descobrir o valor de `hora`:

Reviewer: ```suggestion
Como está de dia, o horário deve estar entre 6 da manhã e meio-dia (12:00), por isso igualamos `hora` a 6. Em seguida, calculamos quantos minuto a mais o ângulo indica. Analisando os casos de teste presentes no enunciado, descobrimos que 0,5° equivale a 2 minutos. Com isso, descobrimos quantos minutos temos dividindo `m` por 0,5 (`tempo = m/0.5`) e depois multiplicando o valor obtido por 2 (`minuto = tempo*2`). Agora resta descobrir o valor de `hora`:
```
